aesara/tensor/basic.py

```diff
@@ -3634,6 +3635,51 @@ def swapaxes(y, axis1, axis2):
     return y.dimshuffle(li)
 
 
+def moveaxis(
+    a: Union[np.ndarray, TensorVariable],
+    source: Union[int, Sequence[int]],
+    destination: Union[int, Sequence[int]],
```

Comment:
I think you might need to use `typing.Sequence` here.  (At least for some versions of Python?)

Reply:
Oh I assumed it came from typing when it autocompleted.